core/domain/app_feedback_report_domain_test.py

```diff
@@ -199,6 +240,21 @@ def test_report_web_platform_validation_fails(self) -> None:
             'Domain objects for web reports have not been implemented yet.'):
             self.web_report_obj.validate()
 
+    def test_raise_not_implemented_error_for_web_platform_feedback(
+        self
+        ) -> None:
+        REPORT_JSON['platform_type'] = 'invalid_platform_type'
+        with self.assertRaisesRegex(
+            NotImplementedError,
+            'Domain objects for web reports must be implemented.'):
+            self.web_report_obj.from_submitted_feedback_dict(REPORT_JSON)
+
+    def test_for_web_platform_feedback(self) -> None:
```

Comment:
Can you please update the test name, according to this format:
test_{{action}}_with_{{with_condition}}_{{has_expected_outcome}}